### PR TITLE
Fix local-bmo script

### PR DESCRIPTION
**Pull request by andfasano**
Sync'up with the latest changes on BMO, fixed the script

### CODE REVIEW

File: metal3-dev/local-bmo.sh [hunk 1]
```diff
@@ -28,6 +28,7 @@ fi
 
 # Stop the machine-api-operator so it does not try to fix the
 # deployment we are going to change.
+cd $SCRIPTDIR
```
Comment (dhellmann) [inline metal3-dev/local-bmo.sh:31]: Is this needed?

Comment (andfasano) [reply metal3-dev/local-bmo.sh:31]: Without didn't work for me

**Pull request merged**